Makefile

```diff
@@ -173,7 +176,7 @@ clean:
 
 .PHONY: distclean 
 distclean: clean
-	$(VE)$(MAKE) -C $(NXDK_DIR)/tools/extract-xiso clean $(QUIET)
+	$(VE)rm -rf $(NXDK_DIR)/tools/extract-xiso/build
```

Comment:
The convention here so far has been to let the sub-project clean up itself (by invoking `disclean` on the associated Makefile) instead of wiping everything away with force, but in this case this approach is ok